.eslintrc-react-test

```diff
@@ -0,0 +1,4 @@
+---
```

Comment:
cc/ @ryan-roemer this all looks good to me! After taking a look at the code I would vote for opener to stay.  I got the impression that it was interfering with test infrastructure.  Do you have any objection to me putting it back in?